Sources/SwiftProtobuf/BinaryDecoder.swift

```diff
@@ -1288,20 +1288,8 @@ internal struct BinaryDecoder: Decoder {
     private mutating func skipOver(tag: FieldTag) throws {
         switch tag.wireFormat {
         case .varint:
-            if available < 1 {
-                throw BinaryDecodingError.truncated
-            }
-            var c = p.load(fromByteOffset: 0, as: UInt8.self)
-            while (c & 0x80) != 0 {
-                p += 1
-                available -= 1
-                if available < 1 {
-                    throw BinaryDecodingError.truncated
-                }
-                c = p.load(fromByteOffset: 0, as: UInt8.self)
-            }
-            p += 1
-            available -= 1
+            // Don't need the value, just ensuring it is validly encoded.
+            let _ = try decodeVarint()
```

Comment:
It's always nice when the more-correct code is also simpler.  ;-)

Reply:
Yea, I started fixing it up to track length and then realized it wasn't worth a second copy that did everything but collect the value.